UEFIExtract/ffsdumper.cpp

```diff
@@ -15,22 +15,52 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 
 #include <fstream>
 
+void FfsDumper::fixFileName(UString &name)
+{
+    // Replace some symbols with underscores for compatibility
+    const char table[] = {
+        '/', // Banned in *nix and Windows
+        '<', '>', ':', '\"', '\\', '|', '?', '*', // Banned in Windows
+    };
+    int nameLength = (int)name.length(); // Note: Qt uses int for whatever reason.
+    if (!nameLength) {
+        name += '_';
```

Comment:
`name = "_";` is more readable as it shows the actual intention of the code — to always provide some filename.

Reply:
fixed in 2926a9ff4625d10989bc2713517a0fe9f395abdf